### ISSUE TITLE
RAM memory getting filled on training

### ISSUE THREAD

**Issue by pnmartinez** [jdb78/pytorch-forecasting#286]
- PyTorch-Forecasting version: 0.8.2
- PyTorch version: 1.7.1
- Python version: 3.7.9
- Operating System: Linux Ubuntu 18.04

### Expected behavior

Following the **N-beats tutorial** on [the docs](https://pytorch-forecasting.readthedocs.io/en/latest/tutorials/ar.html), but with my own dataset.

It's a **large dataset** (+1.5M rows), but I've trimmed it to 100k for dev purposes.

### Actual behavior

**RAM gets filled while training epochs**, even when using `gpus = 1`, leading to a Kernel crash. 

I wonder if, **under the hood, either Pytorch Lightning of Pytorch forecasting** make the classic mistake of `loss += loss`, keeping all the parameters in memory, instead of the adviced `loss += loss.item()`.

**Comment by jdb78** [comment]
To narrow down the problem, do you you have the same issue when running with `log_interval=-1`?

**Comment by pnmartinez** [comment]
Hi,

I am a bit of a newcomer to Pytorch, so can you specify what **method** is this a parameter of? `.fit(log_interval=-1)`?

I've done some googling, but can't seem to find it.

Thanks in advance.
________________________________

**Comment by jdb78** [comment]
It is a NBeats __init__ parameter. So use it in the `from_dataset` method

**Comment by pnmartinez** [comment]
Amazingly, haven't had that problem in a new run of the same notebook today. 

Used `log_interval = -1` when building the net: should it be responsible?

I have one more question: is **Nbeats forecasting** (image below) all the prediction steps at the same time (**multi-step prediction**), or is it doing a **single-step prediction** at every point, and then collecting them for the plot?

**Comment by jdb78** [comment]
It is a multi-step architecture.

Question: If you are using `log_interval > 0`, can you reproduce the memory issue? If yes, this is a bug we need to fix.
